### DISCUSSION TITLE
How to distinguish clicking on overlay and close button.

### DISCUSSION THREAD

**Discussion by bluepenit** [gilbarbara/react-joyride#988] [Q&A]
In v 2.7.2, overlay click action is same as the close button click.
I'd like to pass to the next step when clicking on overlay.
How can I handle this?

**Comment by gilbarbara** [comment]
Hey @bluepenit 
The default behavior for clicking in the overlay is to close the tooltip.
You can either set the `disableOverlayClose` prop or add a `disableBeacon: true` to your steps.

**Comment by bluepenit** [comment]
I have already handled this issue by traditional way.
I have used `disableOverlayClose `and `diableBeacon: true` from joyride origin code.
But does not works.
Thank you for your considering.
